pytest_tests/lib/helpers/grpc_responses.py

```diff
@@ -39,6 +39,7 @@
 EACL_TABLE_IS_NOT_SET = "extended ACL table is not set for this container"
 EACL_NOT_FOUND = "code = 3073.*message = eACL not found"
 EACL_PROHIBITED_TO_MODIFY_SYSTEM_ACCESS = "table validation: it is prohibited to modify system access"
+EACL_CHANGE_PROHIBITED = ".*container ACL is immutable.*"
```

Comment:
this is thrown by CLI, so i suggest to also test behavior with `-f` flag disabling CLI-side pre-checks. Currently, i suppose exception wont be thrown in this case, and IR will deny such request async. Thus, timeout is expected

Reply:
added a check with `-f` flag. Timeout is returned as you expected.